<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cartelLogin">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>50</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>130</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INGRESAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUsuario">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>400</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-bottom: 3px;</string>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
    <property name="placeholderText">
     <string>USUARIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineContrasenia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>400</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-bottom: 3px;</string>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>CONTRASEÃ‘A</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
